src/main/java/org/jenkinsci/test/acceptance/po/Job.java

```diff
@@ -188,10 +209,18 @@ public void copyFile(File file) {
             ZipUtil.pack(file, tmp);
             byte[] archive = IOUtils.toByteArray(new FileInputStream(tmp));
 
-            addShellStep(String.format(
-                    "base64 --decode << ENDOFFILE > archive.zip && unzip -o archive.zip \n%s\nENDOFFILE",
-                    new String(Base64.encodeBase64Chunked(archive))
-            ));
+            if (SystemUtils.IS_OS_WINDOWS) {
+                if (!(controller instanceof LocalController)) {
+                    // TODO: Make it work for RemoteJenkinsController like in Unix (below)
+                    throw new AssumptionViolatedException("Copying files in Windows is only supported if a LocalController is in use. Test will be skipped.");
```

Comment:
http://cygwin.redhat.com  you also need docker and a load of other stuff to run the tests on windows...
see PR#84

What I am saying is that whilst you may not have these tools installed - if they are installed the test should not be skipped. 
Currently I beleive this would be working for me on my setup (I don't think it is the cause of any of the still failing tests that I am seeing at least - although still a number to go through)